MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -34,50 +34,156 @@
 
 @interface MSALPublicClientApplication : NSObject
 
+/*! Whether or not to validate the authority used with the application, default YES */
 @property BOOL validateAuthority;
+
+/*! The authority used with the application object */
 @property (readonly) NSURL *authority;
+
+/*! The client ID of the application */
 @property (readonly) NSString *clientId;
+
+/*! The redirect URI of the application */
 @property (readonly) NSURL *redirectUri;
 
 /*! Used in logging callbacks to identify what component in the application
     called MSAL. */
 @property NSString *component;
 
+/*!
+    Initialize a MSALPublicClientApplication with a given clientID for the default
+    authority
```

Comment:
again, most devs will have no idea what "default authority" means. Either we take the concept of authority out of this function definition altogether, or we briefly summarize the meaning.